test/e2e/utils/utils.go

```diff
@@ -40,6 +40,19 @@ const (
 	poll = time.Second
 )
 
+// PrintPodLogsAndFailIfErrorExists prints the log of the pod that did not run successfully and error message if the provided error is not nil.
+func PrintPodLogsAndFailIfErrorExists(errPodNotSuccess error, ctx context.Context, f *framework.Framework, podName, containerName string) {
+	if errPodNotSuccess != nil {
+		log, err := e2epod.GetPodLogs(ctx, f.ClientSet, f.Namespace.Name, podName, containerName)
+		if err != nil {
+			framework.Failf("unable to get log from pod: %v", err)
```

Comment:
This is when we cannot get pod logs. If we get the logs, it will log both logs and the `errPodNotSuccess`.
Do you think it makes more sense to have `framework.Logf("unable to get log from pod: %v", err)`